lib/src/object_group.dart

```diff
@@ -1,32 +1,32 @@
-part of tiled;
+import 'package:xml/xml.dart';
+import 'tile_map_parser.dart';
+import 'tile_map.dart';
+import 'tmx_object.dart';
+import 'node_dsl.dart';
 
 class ObjectGroup {
-  String name;
-  String color;
+  String name= "";
+  String color = "";
```

Comment:
What does it mean for these to default to empty strings?

Reply:
I don't know, null is not much better. ;-)